Sample/Kitchen Sink/DemonstrationTools/CrossWindCalculator.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Shape+Extensions.swift
```

Comment:
Looks like someone round here's been stealing some comments 😂 😂

Reply:
Ha, the pList was not drawing the correct comment, I think its so weirdness with the app within package sort thing